### DISCUSSION TITLE
新增数据时，原有拖拽节点位置及画布位置会重新布局，有什么办法吗？

### DISCUSSION THREAD

**Discussion by 0xRelter** [antvis/Graphin#479] [Q&A]
新增数据时，原有拖拽节点位置及画布位置会重新布局，有什么办法吗？

**Comment by pomelo-nwu** [comment]
没有太好的办法，因为新增的节点如果不改变原有的布局，可能会导致节点和边重叠，现在渐进力导的思路是尽可能减少位置的变化，但是还是会变化的。
